Release/include/cpprest/http_msg.h

```diff
@@ -548,6 +550,60 @@ class http_response
     /// <returns>A new HTTP response.</returns>
     http_response(http::status_code code) : _m_impl(std::make_shared<details::_http_response>(code)) {}
 
+    /// <summary>
+    /// Constructs a response object
+    /// </summary>
+    /// <returns>A new HTTP response.</returns>
+    http_response(const http_response& _Other) : _m_impl(_Other._m_impl) {}
+
+    /// <summary>
+    /// Constructs a response object
+    /// </summary>
+    /// <returns>A new HTTP response.</returns>
+    http_response(http_response&& _Other) : _m_impl(std::move(_Other._m_impl)) {}
+
+    /// <summary>Replaces the contents of one <c>http_request</c> object with another.</summary>
+    /// <param name="_Other">The source <c>http_request</c> object.</param>
+    /// <remarks>
+    /// As <c>http_request</c> behaves like a smart pointer, after a copy assignment, this <c>http_request</c>
+    /// objects represents the same actual http_request as <paramref name="_Other"/> does.
+    /// </remarks>
+    /// <returns>A new HTTP response.</returns>
+    http_response& operator=(const http_response& _Other)
+    {
+        if (this != &_Other)
+        {
+            _m_impl = _Other._m_impl;
+        }
+        return *this;
+    }
+
+    /// <summary>
+    /// Destructor frees any held resources.
+    /// </summary>
+    ~http_response() = default;
+
+    /// <summary>Replaces the contents of one <c>http_request</c> object with another.</summary>
+    /// <param name="_Other">The source <c>http_request</c> object.</param>
+    /// <remarks>
+    /// As <c>http_request</c> behaves like a smart pointer, after a copy assignment, this <c>http_request</c>
+    /// objects represents the same actual http_request as <paramref name="_Other"/> does.
+    /// </remarks>
+    /// <returns>A new HTTP response.</returns>
+    http_response& operator=(http_response&& _Other)
+    {
+        if (this != &_Other)
+        {
+            _m_impl = std::move(_Other._m_impl);
+        }
+        return *this;
+    }
+
+    /// <summary>
+    /// Destructor frees any held resources.
+    /// </summary>
+    ~http_response() = default;
```

Comment:
These don't appear to do anything over just accepting the compiler's defaults. The original class has an empty destructor, which is somewhat redundant, not an excuse to add even more redundant stuff